iap_watcher/iap_watcher_test.py

```diff
@@ -34,6 +34,7 @@ def testEnable(self):
         'iap_metadata_key': 'AEF_IAP_state',
         'output_state_file': path,
         'polling_interval': 1,
+        'fetch_keys': False,
```

Comment:
test for true case, please

Reply:
Done. Warning: non-hermetic. Could try to mock os.system as a compromise if that's an issue. (Mock package apparently not available in the testing container...maybe could install, but that's also non-hermetic.)

Reply:
Ah that's a shame. I'd prefer to mock or not include the test, then, we wouldn't want non-hermetic tests here.

Reply:
Rolled a somewhat ugly one-off mock, but probably better than no test, PTAL.